examples/canoniCAI/documentation/how to add a competency in canoniCAI.md

```diff
@@ -0,0 +1,308 @@
+# CanoniCAI
```

Comment:
Let's not include space in file names. So instead of `how to add a competency in canoniCAI.md` it can just be `add_competency.md`

Reply:
will fix this now